src/Runner.Worker/FileCommandManager.cs

```diff
@@ -259,4 +259,99 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
             return text.Substring(originalIndex, lfIndex - originalIndex);
         }
     }
+
+    public sealed class SetStepSummaryCommand : RunnerService, IFileCommandExtension
+    {
+        public string ContextName => "step_summary";
+        public string FilePrefix => "step_summary_";
+
+        private const int AttachmentSizeLimit = 128 * 1024;
```

Comment:
`private const int _attachmentSizeLimit = 128 * 1024;` and move it up before all `public string xxx`